### PR TITLE
AIRNEIS-45 - feat(homepage): Creation of category cards and home page

### PR DESCRIPTION
**Pull request by KyDev25**
# DONE 

- Home page for web with desktop.

# TEST

- Add categories
- Make dev
- Enjoy !

# STILL TO DO

- Test if everything is working correctly, because of environmental problems I can't test directly on my computer.

### CODE REVIEW

File: front/src/pages/index.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    }
+  }
+}
+const Home = props => {
+  const { t } = useTranslation()
+  const { categories } = props
+
+  return (
+    <Page className="h-screen text-secondary">
+      <div className="w-full h-1/4 sm:h-1/3 bg-[url('https://picsum.photos/1800/350')] bg-center flex items-center justify-center">
```
Comment (barnabepilliaudin) [inline front/src/pages/index.jsx]: Don't put a lorem picsum url but try to have that will be the same all the time.

File: front/src/utils/config.js [hunk 2]
```diff
@@ -12,6 +12,10 @@ const config = {
     productCard: {
       width: 350,
       height: 200
+    },
+    categoryCard: {
+      width: 350,
```
Comment (barnabepilliaudin) [inline front/src/utils/config.js]: Please rename to imageWidth
Same for height to imageHeight
Can you please also do it for product card (don't forget to modify the component)

File: front/src/components/CategoryCard.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+const CategoryCard = ({ category, className, otherProps }) => {
+  const router = useRouter()
+  const goToCategoryPage = () => router.push(apiRoutes.categoryId(category.ID))
+
+  return (
+    <li className={clsx(className, "flex flex-col gap-1 max-w-md  rounded mx-auto relative")} onClick={goToCategoryPage} {...otherProps}>
+      <Image
+        alt={category.name}
+        src={category.thumbnailURL}
+        width={config.images.categoryCard.width}
```
Comment (barnabepilliaudin) [inline front/src/components/CategoryCard.jsx]: See comment for config.js

**Review by BarnabePILLIAUDIN — CHANGES_REQUESTED** [review]
Please consider the following changes.

Welcome to the dev team

**Comment by BarnabePILLIAUDIN** [comment]
@KyDev25 Also can you please rebase staging as your branch is behind

**Comment by BarnabePILLIAUDIN** [comment]
image

Check the category page It may be broken

File: front/src/pages/index.jsx (hunk 1 above)
Comment (KyDev25) [reply front/src/pages/index.jsx]: Done!

File: front/src/components/CategoryCard.jsx (hunk 3 above)
Comment (KyDev25) [reply front/src/components/CategoryCard.jsx]: Done !

File: front/src/utils/config.js (hunk 2 above)
Comment (KyDev25) [reply front/src/utils/config.js]: Than you for advice! Done!

File: front/src/pages/index.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    }
+  }
+}
+const Home = props => {
+  const { t } = useTranslation()
+  const { categories } = props
+
+  return (
+    <Page className="h-screen text-secondary">
+      <div className="w-full h-1/4 sm:h-1/3 bg-[url('https://imgs.search.brave.com/ZE_b8CzcWBoSZU5RW7btiybrw1zko5bsTjUxgo3MT1o/rs:fit:860:0:0/g:ce/aHR0cHM6Ly9mYWMu/aW1nLnBtZHN0YXRp/Yy5uZXQvc2NhbGUv/aHR0cC4zQS4yRi4y/RnByZDItYm9uZS1p/bWFnZS4yRXMzLXdl/YnNpdGUtZXUtd2Vz/dC0xLjJFYW1hem9u/YXdzLjJFY29tLjJG/RkFDLjJGMjAxOS4y/RjA5LjJGMjMuMkYw/YzQyMDU2Yi1lNjYx/LTQ2MmItODhiYi00/MWM3OTRlOTkyMjQu/MkVqcGVnL2F1dG94/NjAwL3F1YWxpdHkv/NjUvZ3JhbmQtY2Fu/YXBlLWRlLWphcmRp/bi5qcGc')] bg-center flex items-center justify-center">
```
Comment (cedricgautier) [inline front/src/pages/index.jsx:23]: We should change this...later.

**Review by cedricgautier — DISMISSED** [review]

**Review by BarnabePILLIAUDIN — APPROVED** [review]
Ok for merge but we really need to rework the image

**Pull request merged**